app/main.py

```diff
@@ -1 +1,23 @@
-# write your code here
+from app.cafe import Cafe
+from app.errors import (
+    VaccineError,
+    NotWearingMaskError
+)
+
+
+def go_to_cafe(friends: list, cafe: Cafe) -> str:
+    count_of_non_vaccinated = 0
+    count_of_musks = 0
+    for friend in friends:
+        try:
+            cafe.visit_cafe(friend)
+        except (VaccineError):
+            count_of_non_vaccinated += 1
```

Comment:
```suggestion
            return "All friends should be vaccinated"
```
no need to create `count_of_non_vaccinated ` variable